src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -99,14 +99,14 @@ object SbtBundle extends AutoPlugin {
     val configFile = writeConfig(configTarget, bundleConf.value)
     val bundleMappings =
       configFile.pair(relativeTo(configTarget)) ++ (mappings in bundleTypeConfig).value.map(relParent)
-    val tgz = Archives.makeTgz(bundleTarget, (packageName in Universal).value, bundleMappings)
-    val tgzName = tgz.getName
-    val exti = tgzName.lastIndexOf('.')
-    val hash = Hash.toHex(digestFile(tgz))
-    val hashName = tgzName.take(exti) + "-" + hash + tgzName.drop(exti)
-    val hashTgz = tgz.getParentFile / hashName
-    IO.move(tgz, hashTgz)
-    hashTgz
+    val archive = Archives.makeZip(bundleTarget, (packageName in Universal).value, bundleMappings)
```

Comment:
Reading a comment from sat-native-packager's `ZipHelper`:
* Note: This is known to have some odd issues on MacOSX whereby executable permissions
* are not actually discovered, even though the Info-Zip headers exist and work on
* many variants of linux.  Yay Apple.
Excited to see whether this issue shows up ...